distributed/http/scheduler/prometheus/core.py

```diff
@@ -102,6 +102,12 @@ def collect(self) -> Iterator[GaugeMetricFamily | CounterMetricFamily]:
                 tasks.add_metric([state], task_counter.get(state, 0.0))
         yield tasks
 
+        yield GaugeMetricFamily(
```

Comment:
considering that we're never deleting/forgetting any task groups this could/should be a Counter.

Reply:
AFAIK we do delete task groups:

https://github.com/dask/distributed/blob/90ee0a8bb6450449770ad98b12f0397be1ae1c60/distributed/scheduler.py#L2031-L2035

Reply:
ah, interesting. never midn then